app/views/shared/_sidebar.html.erb

```diff
@@ -34,7 +34,7 @@
         <ul class="menu">
           <%= render "/shared/sidebar_content/dashboard" unless authorized_user.can_advertise_and_publish? %>
           <%= render "/shared/sidebar_content/dashboards" if authorized_user.can_advertise_and_publish? %>
-          <%#= render "/shared/sidebar_content/inbox" if authorized_user.can_view_emails? %>
+          <%= render "/shared/sidebar_content/inbox" if authorized_user.can_view_emails? %>
```

Comment:
should this only be for admins?

Reply:
It's only for authorized users that can view emails. I did it this way so we can add more rules in the future.

Reply:
Ah didn't see that we already had an authorizer.